src/core/agent/services/ipexCommunicationService.test.ts

```diff
@@ -892,41 +924,111 @@ describe("Ipex communication service of agent", () => {
         },
       },
     ]);
-    credentialStorage.getCredentialMetadata = jest.fn().mockResolvedValue({
-      id: "id",
-    });
+    credentialStorage.getCredentialMetadata = jest.fn().mockResolvedValue(null);
+    multisigService.multisigAdmit = jest
+      .fn()
+      .mockResolvedValue({ op: { name: "opName", done: true } });
+
+    notificationStorage.findAllByQuery = jest.fn().mockResolvedValue([
+      {
+        type: "NotificationRecord",
+        id: id,
+        createdAt: new Date(),
+        a: {
+          r: "/exn/ipex/grant",
+          d: "EIDUavcmyHBseNZAdAHR3SF8QMfX1kSJ3Ct0OqS0-HCW",
+        },
+        route: "/exn/ipex/grant",
+        read: true,
+        linkedGroupRequests: {
+          "EDm8iNyZ9I3P93jb0lFtL6DJD-4Mtd2zw1ADFOoEQAqw": false,
+        },
+        connectionId: "EEFjBBDcUM2IWpNF7OclCme_bE76yKE3hzULLzTOFE8E",
+        updatedAt: new Date(),
+      },
+    ]);
+
+    await ipexCommunicationService.acceptAcdc(id);
 
-    await ipexCommunicationService.acceptAcdcFromMultisigExn(id);
-    expect(Agent.agent.multiSigs.multisigAdmit).toBeCalledTimes(1);
-    expect(credentialStorage.saveCredentialMetadataRecord).toBeCalledWith({
-      id: "EJvvnAZruVSfvPZjzGwyTR3RQApoK7228du0c8flDcaF",
-      isArchived: false,
-      credentialType: "title",
-      issuanceDate: new Date("2024-07-31T02:45:16.860000+00:00").toISOString(),
-      connectionId: "EKhebhdg6jOqK7ZgY-cFpx6rozpUave8llE2B15ioNHi",
-      schema: "EBfdlu8R27Fbx-ehrqwImnK-8Cm79sqbAQ4MmvEAYqao",
-      status: CredentialStatus.PENDING,
-    });
     expect(operationPendingStorage.save).toBeCalledWith({
       id: "opName",
       recordType: OperationPendingRecordType.ExchangeReceiveCredential,
     });
     expect(
-      Agent.agent.signifyNotifications.addPendingOperationToQueue
+      signifyNotificationService.addPendingOperationToQueue
     ).toBeCalledTimes(1);
-    expect(deleteNotificationMock).toBeCalledWith(id);
   });
 
-  test("cannot accept ACDC from multisig exn if the notification is missing in the DB", async () => {
```

Comment:
Why was this test removed? We have a `IpexCommunicationService.NOTIFICATION_NOT_FOUND` check that needs testing